docs/topics/snippets.rst

```diff
@@ -229,3 +229,25 @@ Adding tags to snippets is very similar to adding tags to pages. The only differ
         ]
 
 The :ref:`documentation on tagging pages <tagging>` has more information on how to use tags in views.
+
+
+ClusterableModel snippets
+-------------------------
+
+If you want to group a snippet you can use ClusterableModel. For example you want to insert bands and there members.
```

Comment:
minor grammar / typos.
Should be ->

If you want to group a Snippet you can use ``ClusterableModel``. For example you want to insert bands and their members.

there -> their above